persist-cli-tests/src/test/resources/test-src/output/tool_test_migrate_1/persist/migrations/2023-05-09-09-01-19_migrationLabel/script.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE MedicalObject (
+    objectId INT PRIMARY KEY,
+    objectName VARCHAR(191),
+    types VARCHAR(191),
+    objectFlag BOOLEAN
+);
+CREATE TABLE MedicalTest (
+    testId VARCHAR(191) PRIMARY KEY,
+    object1ObjectId INT
+);
+ALTER TABLE MedicalNeed
+ADD COLUMN description VARCHAR(191);
```

Comment:
Can you add new line in between

Reply:
@daneshk where exactly should I add the new line? after the create table?